docs/configuration/integrations/email.md

```diff
@@ -0,0 +1,28 @@
+# Emails
+
+## SMTP Configuration
+
+Follow the instructions below to set up the Codacy integration with SMTP to send emails:
+
+1.  Edit the file `values-production.yaml`, set `global.email.enabled: "true"` and define the remaining values with the credentials for your smtp server:
```

Comment:
Is the YAML block supposed to be already included in the `values-production.yaml` file? :thinking: It seems it's not there:

https://github.com/codacy/chart/blob/move-email-config-to-file/codacy/values-production.yaml

Reply:
forgot that part